source/operations/checklists/hardware.rst

```diff
@@ -202,16 +208,85 @@ Storage
 
 .. cond:: not k8s
 
+   Recommended Storage Mediums
+   +++++++++++++++++++++++++++
+
    MinIO recommends using flash-based storage (NVMe or SSD) for all workload types and scales.
    Workloads that require high performance should prefer NVMe over SSD.
 
    MinIO deployments using HDD-based storage are best suited as cold-tier targets for :ref:`Object Transition ("Tiering") <minio-lifecycle-management-tiering>` of aged data.
    HDD storage typically does not provide the necessary performance to meet the expectations of modern workloads, and any cost efficiencies at scale are offset by the performance constraints of the medium. 
 
-   Format Drives as XFS
-   ++++++++++++++++++++
+   Use Direct-Attached "Local" Storage (DAS)
+   +++++++++++++++++++++++++++++++++++++++++
+
+   DAS-storage, such as locally-attached JBOD (Just a Bunch of Disk) arrays, provide significant performance and consistency advantages over networked (NAS, SAN, NFS) storage.
```

Comment:
Is `DAS-storage` common usage? To me, it reads as `Direct-Attached Storage-Storage`